lib/prefab/config_client.rb

```diff
@@ -176,10 +176,10 @@ def load_configs(configs, source)
       finish_init!(source, project_id)
     end
 
-
     def cache_path
       return @cache_path unless @cache_path.nil?
-      @cache_path ||= File.join(Dir.home, '.prefab',".prefab.cache.#{@base_client.options.api_key_id}.json")
+
+      @cache_path ||= File.join(ENV.fetch('XDG_CACHE_HOME', File.join(Dir.home, '.cache')), ".prefab.cache.#{@base_client.options.api_key_id}.json")
```

Comment:
i've read this a couple times and i'm still not sure what the path here is when there is or is not an XDG_CACHE_HOME - can this be
1) less terse
2) log the cache location the first time its generated

Reply:
mm. Test is more readable https://github.com/prefab-cloud/prefab-cloud-ruby/pull/148/files#diff-0316acc4f7bad770d459a6870363ba6411b13e6ad04a2beef589b767894d0af0R101 but yeah, a bit of a long line. I can tweak it.